app/models/person.rb

```diff
@@ -36,8 +39,7 @@ class Person < ApplicationRecord
   accepts_nested_attributes_for :location,
     reject_if: ->(attributes) { attributes["city_town"].blank? }
   has_many :matches # , dependent: :destroy
-
-  has_one :user, dependent: :destroy
+  belongs_to :user, optional: true
 
```

Comment:
I think we need to add a validation like this to prevent multiple person records under a given Org scope having the same `user_id`

`validates :user_id, uniqueness: { scope: :organization_id }, allow_nil: true`